plugins/inventory/proxmox.py

```diff
@@ -69,6 +69,24 @@
         description: Gather LXC/QEMU configuration facts.
         default: no
         type: bool
+      strict:
+        description:
+          - If yes make invalid entries a fatal error, otherwise skip and continue.
```

Comment:
These should come from the `constructed` docs fragment, like here: https://github.com/ansible-collections/community.general/blob/main/plugins/inventory/virtualbox.py#L16

What you need to keep is
```.yaml
    option_name:
        version_added: 2.5.0
```
for all these options though, so that it is clear that these have been added to this plugin.

Reply:
Yep, looks good!